src/components/about/About.js

```diff
@@ -0,0 +1,42 @@
+import React from "react"
+import "./about.scss"
+
+const About = props => {
```

Comment:
It's actually fine to replace `props` with `()` since we're not using them in this component - it communicates that fact to everyone else.

Reply:
> Decided to leave in `props` for the time being, can discuss later.